io.typora.editor.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "io.typora.editor",
+  "command": "/app/start.sh",
+  "separate-locales": false,
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=host",
+    "--share=ipc",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
```

Comment:
This is a nitpick but there's no need to nest modules, `pandoc` and `typora` could be directly here.